docs/metrics.md

```diff
@@ -0,0 +1,587 @@
+
+[Cardinality
+limit](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#cardinality-limits)
+is a throttling mechanism which allows the metrics collection system to have a
+predictable and reliable behavior when there is a cardinality explosion, be it
+due to a malicious attack or developer making mistakes while writing code.
+
+OpenTelemetry has a default cardinality limit of `2000` per metric. This limit
+can be configured at the individual metric level using the [View
+API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view)
+leveraging the
+[`cardinality_limit`](https://docs.rs/opentelemetry_sdk/latest/opentelemetry_sdk/metrics/struct.Stream.html#structfield.cardinality_limit)
+setting.
+
+It's important to understand that this cardinality limit applies only at the
+OpenTelemetry SDK level, not to the ultimate cardinality of the metric as seen
+by the backend system. For example, while a single process might be limited to
+2000 attribute combinations per metric, the actual backend metrics system might
+see much higher cardinality due to:
+
+1. Resource attributes (such as `service.instance.id`, `host.name`, etc.) that
+   can be added to each metric
+2. Multiple process instances running the same application across your
+   infrastructure
+3. The possibility of reporting different key-value pair combinations in each
+   export interval, as the cardinality limit only applies to the number of
+   unique attribute combinations tracked during a single export interval. (This
+   is only applicable to Delta temporality)
+
+Therefore, the actual cardinality in your metrics backend can be orders of
+magnitude higher than what any single OTel SDK process handles in an export
+cycle.
+
+#### Cardinality Limits - Implications
+
+Cardinality limits are enforced for each export interval, meaning the metrics
+aggregation system only allows up to the configured cardinality limit of unique
+attribute combinations per metric. Understanding how this works in practice is
+important:
+
+* **Cardinality Capping**: When the limit is reached within an export interval,
+  any new attribute combination is not individually tracked but instead folded
+  into a single aggregation with the attribute `{"otel.metric.overflow": true}`.
+  This preserves the overall accuracy of aggregates (such as Sum, Count, etc.)
+  even though information about specific attribute combinations is lost. Every
+  measurement is accounted for - either with its original attributes or within
+  the overflow bucket.
+
+* **Temporality Effects**: The impact of cardinality limits differs based on the
+  temporality mode:
+  
+  * **Delta Temporality**: The SDK "forgets" the state after each
+    collection/export cycle. This means in each new interval, the SDK can track
+    up to the cardinality limit of completely different attribute combinations.
```

Comment:
Rephrasing suggestion:

"the SDK can track as many unique attribute combinations as the metric's cardinality limit."

Reply:
I see that you have used "distinct" in the text below. It'd be nice to stay consistent with the choice of word. "distinct" vs "unique"

"the SDK can track as many distinct attribute combinations as the metric's cardinality limit."